src/actions.js

```diff
@@ -230,6 +326,8 @@ export function evaluateCell(cellId) {
       dispatch(evaluateResourceCell(cell))
     } else if (cell.cellType === 'css') {
       dispatch(evaluateCSSCell(cell))
+    } else if (cell.cellType === 'plugin') {
+      dispatch(evaluateLanguagePluginCell(cell))
```

Comment:
I'd anticipate we'll have more plugin types than just language plugins (output plugins, for example).  I don't think we want a proliferation of cell types for each type of plugin.  Should we add an `evaluatePluginCell` function that dispatches to different known plugin types based on the content of the JSON?  I.e. add a field "pluginType" to the JSON that would be "language" in this case, but might be "output", etc.?  Maybe we want to move the rest of the payload under a "settings" key in that case, because it will be easier to validate against a JSON Schema that way (but that's fine if not).

Reply:
yeah i think that's right. minimally, we already know we want to support MD extensions. I'll go make that change.